libs/island-ui/core/src/lib/AsyncSearch/shared/Label/Label.tsx

```diff
@@ -9,10 +9,11 @@ export interface LableProps extends LabelHTMLAttributes<HTMLLabelElement> {
 }
 
 export const Label = forwardRef<HTMLLabelElement, LableProps>(
-  ({ hasError, color, children, ...props }) => {
```

Comment:
Is this `ref` never used or am I missing something 🤔

Reply:
I image this is never used but the forwardRef was there initially without a ref. I could also just remove the HOC but this felt a little safer.

Reply:
@oddsson can you re review or approve if you are satisfied :)